src/api/entitlements.js

```diff
@@ -217,8 +227,10 @@ export class Entitlement {
     }
     // Wildcard allows this product.
     const eq = product.indexOf(':');
-    if (eq != -1 &&
-        this.products.includes(product.substring(0, eq + 1) + '*')) {
+    if (
```

Comment:
I would prefer the previous formatting.

Reply:
This formatting was automatically done by `prettier`